scene/gui/line_edit.cpp

```diff
@@ -358,6 +358,12 @@ void LineEdit::_gui_input(Ref<InputEvent> p_event) {
 					FALLTHROUGH;
 				}
 				case KEY_LEFT: {
+					if (selection.enabled && !k->get_shift()) {
+						set_cursor_position(selection.begin);
+						deselect();
+						handled = true;
+						break;
+					}
```

Comment:
This needs to be moved down a couple lines, above the `shift_selection_check_pre` call to keep consistency with Apple style keys. Will need to add braces to the `#ifndef ` if.

Reply:
Will do.  I happen to have a Mac nearby and can test this out there.

Reply:
Note, it seems from the code pattern that a similar change needed to occur for the KEY_RIGHT case so I went ahead and did that too.